docs/GetStarted/getting_started_linux_cmake.md

```diff
@@ -57,9 +61,9 @@ export CXX=clang++
 Clone the repository and initialize its submodules:
 
 ```shell
-$ git clone https://github.com/google/iree.git
```

Comment:
I like the leading `$` since it lets us differentiate between commands and comments (`# comment`).

Unfortunately, GitHub flavored markdown and our current website theming don't exclude the leading `$` when you select / copy text from code blocks, so I'd be okay with removing them. If we remove them here, we should also remove them from the other docs in the `GetStarted/` directory.